helpers/constants.js

```diff
@@ -0,0 +1,66 @@
+export default {
```

Comment:
#### Nitpick

- Is everything in here relevant to the product?

Reply:
Resolved